cmd/stencil/stencil.go

```diff
@@ -95,11 +108,6 @@ func main() {
 			Name:  "dev",
 			Usage: "Use local manifests instead of remote ones, useful for development",
 		},
-		&cli.StringFlag{
-			Name:    "github-ssh-key",
-			Usage:   "SSH Key to use to download templates with, if not set ~/.ssh/config is read and falls back to ssh-agent",
-			EnvVars: []string{"GITHUB_SSH_KEY"},
-		},
```

Comment:
SSH support didn't make much sense when extensions themselves _have_ to be downloaded from Github because of the usage of the Github API. So, it's gone now, and we use the GH cli to dish out the token if not supplied.